test/e2e/tests/test_keyspace.py

```diff
@@ -72,16 +72,17 @@ def keyspace_basic():
     try:
         _, deleted = k8s.delete_custom_resource(ref, wait_periods=3, period_length=10)
         assert deleted
+        time.sleep(DELETE_WAIT_AFTER_SECONDS)
     except:
         pass
-
```

Comment:
We need a wait until keyspace deleted call here.. but we can add it in the next PR